addons/web/static/src/search/utils/dates.js

```diff
@@ -1,8 +1,9 @@
 /** @odoo-module **/
 
-import { _lt } from "@web/core/l10n/translation";
 import { Domain } from "@web/core/domain";
+import { serializeDate, serializeDateTime } from "@web/core/l10n/dates";
 import { localization } from "@web/core/l10n/localization";
+import { _lt } from "@web/core/l10n/translation";
```

Comment:
not necessary to move this line, but I'm OK with that

Reply:
My bad: I activated the "organize imports on save" feature on my IDE, will revert this